pages/docs/couple-your-code/couple-your-code-defining-mesh-connectivity.md

```diff
@@ -61,3 +67,30 @@ if(dim==3)
 [...]
 
 ```
+
+## Changes in v3
+
+Version 3 overhauls the definition of meshes.
+
+Connectivity now consists of explicitly defined elements (via the API) and implicitly defined elements.
+preCICE automatically generates all implicit elements.
+As an example, explicitly defining a triangle ABC via the API guarantees the existence of the implicit edges AB, AC, and BC.
+
+Furthermore, all connectivity is defined via vertex IDs only. There are no more edge IDs to worry about.
+The order of vertices also doesn't matter. Triangles BAC and CAB are considered duplicates.
+
+```cpp
```

Comment:
A sentence introducing this list (or an explicit connection to the previous one) is missing.

Reply:
```suggestion
The API for defining individual connectivity elements looks as follows:

```cpp
```